modules/AOR_Reports/AOR_Report.php

```diff
@@ -903,7 +903,7 @@ public function getTotalHTML($fields, $totals)
         $currency->retrieve($GLOBALS['current_user']->getPreference('currency'));
 
         $showTotal = false;
-        $html = '';
+        $html = '</table><table>';
```

Comment:
This change is causing the tests to fail. Can you please update the test?
```
1) AOR_ReportTest: Testget total html
 Test  tests/unit/modules/AOR_Reports/AOR_ReportTest.php:testgetTotalHTML
Failed asserting that two strings are identical.
- Expected | + Actual
@@ @@
-<thead class='fc-head'><tr><th>total Sum</td></tr></thead></body><tr class='oddListRowS1'><td>60</td></tr></body>
+</table><table><thead class='fc-head'><tr><th>total Sum</th></tr></thead><tbody><tr class='oddListRowS1'><td>60</td></tr></tbody>
#1  /home/travis/build/salesagility/SuiteCRM/tests/unit/modules/AOR_Reports/AOR_ReportTest.php:272
#2  AOR_ReportTest->testgetTotalHTML
--
```

I think adding the closing table element with the opening table is not an appropriate solution. I think these table elements should be in the build_report_html method instead. That way getTotalHTML method, will output valid or a complete set of tags. Plus it would be more clear to developers. Please removing this line and placing the closing element before the call to getTotalHTML (line 819) and the opening table element after the call to getTotalHTML ( line 821)

Reply:
Hey @daniel-samson , thanks for suggestions. I agree with you to that adding closing table element with opening table element is not an ideal solution. I read your suggestions. But I don't think we should place these table elements in the `build_report_html` method. 

Instead, I propose another better way. Before calling `getTotalHTML', we'll close the table tag and inside `getTotalHTML` method, we'll open the table tag and in the end of that same method, we'll close the table tag. So, all HTML content related to `getTotalHTML` will stays separate from main method. Kindly let me know, what you think about this.

Reply:
Hi @ChangezKhan, I think that would a more appropriate solution. We just need update the closing table element in build_report_html. Can you please ensure that both methods output "completed" elements.

Please can you also update the test to so that the expected matches the output of getTotalHTML?